### PR TITLE
Set up server routes + change axios calls

### PR DESCRIPTION
**Pull request by elocowan**
Integrate front end with local host and respond to axios calls with
dummy data where appropriate. Preparing front end for integration with
database.

### CODE REVIEW

File: client/src/components/AddReview.jsx [hunk 1]
```diff
@@ -365,8 +365,8 @@ class AddReview extends React.Component {
             </div>
             <div className="row">
               {this.props.characteristics
-                ? this.props.characteristics.map((type) => (
-                  <div className="col-4 my-3 px-3">
+                ? this.props.characteristics.map((type, index) => (
+                  <div key={index} className="col-4 my-3 px-3">
```
Comment (AngieD8615) [inline client/src/components/AddReview.jsx:369]: see note about key={index}

File: client/src/components/IndividualReview.jsx [hunk 2]
```diff
@@ -31,8 +31,8 @@ const IndividualReview = ({
         {photos.length > 0
           ? (
             <div className="col d-flex justify-content-around">
-              {photos.map((picture) => (
-                <img src={picture.url} alt="Can not be displayed" width="50" height="50" style={{ objectFit: 'cover' }} />
+              {photos.map((picture, index) => (
+                <img key={index} src={picture.url} alt="Can not be displayed" width="50" height="50" style={{ objectFit: 'cover' }} />
```
Comment (AngieD8615) [inline client/src/components/IndividualReview.jsx:35]: see note about key={index}

File: client/src/components/ProductBreakdown.jsx [hunk 3]
```diff
@@ -3,12 +3,12 @@ import React from 'react';
...
   <div className="col no-gutters">
     {props.data
-      ? props.data.map((type) => {
+      ? props.data.map((type, index) => {
         const category = type[0];
         const oldNumber = type[1].value;
         const number = (oldNumber / 5) * 100;
         return (
-          <div className="row no-gutters mb-2">
+          <div key={index} className="row no-gutters mb-2">
```
Comment (AngieD8615) [inline client/src/components/ProductBreakdown.jsx:11]: see note about key={index}

**Review by AngieD8615 — APPROVED** [review]

File: client/src/components/ProductBreakdown.jsx (hunk 3 above)
Comment (elocowan) [reply client/src/components/ProductBreakdown.jsx:11]: thanks for the pointer. I've made the appropriate changes 👍

**Pull request merged**